### ISSUE TITLE
Fix Terraform Warnings

### ISSUE THREAD

**Issue by mcapuccini** [kubenow/KubeNow#29]
This doesn't have big priority, but we should fix it a certain point, to avoid errors in the next Terraform versions.

```bash
Warnings:

  * template_file.master_bootstrap: using template_file as a resource is deprecated; consider using the data source instead
  * template_file.node_bootstrap: using template_file as a resource is deprecated; consider using the data source instead
  * template_file.edge_bootstrap: using template_file as a resource is deprecated; consider using the data source instead

No errors found. Continuing with 3 warning(s).
```

**Comment by mcapuccini** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix Terraform Warnings](https://trello.com/c/3LhwPwOf/7-fix-terraform-warnings)

**Comment by mcapuccini** [comment]
Fixed in #160
